src/test/java/com/epam/tc/hw3/tests/TestHomePageControlElements.java

```diff
@@ -1,11 +1,8 @@
 package com.epam.tc.hw3.tests;
 
 
-import com.epam.tc.hw3.pages.HomePageObject;
 import com.epam.tc.hw3.utils.PagesDataProviders;
```

Comment:
this change should not be a part of this PR

Reply:
Done